specification/notificationhubs/Notificationhubs.Management/models.tsp

```diff
@@ -0,0 +1,1600 @@
+  @visibility(Lifecycle.Read, Lifecycle.Create)
+  name?: string;
+
+  /**
+   * Gets or sets the RegistrationTtl of the created NotificationHub
+   */
+  registrationTtl?: string;
+
+  /**
+   * Gets or sets the AuthorizationRules of the created NotificationHub
+   */
+  @visibility(Lifecycle.Read)
+  @OpenAPI.extension("x-ms-identifiers", #["keyName"])
+  authorizationRules?: SharedAccessAuthorizationRuleProperties[];
+
+  /**
+   * Description of a NotificationHub ApnsCredential.
+   */
+  apnsCredential?: ApnsCredential;
+
+  /**
+   * Description of a NotificationHub WnsCredential.
+   */
+  wnsCredential?: WnsCredential;
+
+  /**
+   * Description of a NotificationHub GcmCredential.
+   */
+  gcmCredential?: GcmCredential;
+
+  /**
+   * Description of a NotificationHub MpnsCredential.
+   */
+  mpnsCredential?: MpnsCredential;
+
+  /**
+   * Description of a NotificationHub AdmCredential.
+   */
+  admCredential?: AdmCredential;
+
+  /**
+   * Description of a NotificationHub BaiduCredential.
+   */
+  baiduCredential?: BaiduCredential;
+
+  /**
+   * Description of a NotificationHub BrowserCredential.
+   */
+  browserCredential?: BrowserCredential;
+
+  /**
+   * Description of a NotificationHub XiaomiCredential.
+   */
+  xiaomiCredential?: XiaomiCredential;
+
+  /**
+   * Description of a NotificationHub FcmV1Credential.
+   */
+  fcmV1Credential?: FcmV1Credential;
+
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  @visibility(Lifecycle.Read)
+  dailyMaxActiveDevices?: int64;
+}
+
+/**
+ * SharedAccessAuthorizationRule properties.
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-provisioning-state" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+model SharedAccessAuthorizationRuleProperties {
+  /**
+   * Gets or sets the rights associated with the rule.
+   */
+  rights: AccessRights[];
+
+  /**
+   * Gets a base64-encoded 256-bit primary key for signing and
+   * validating the SAS token.
+   */
+  @secret
+  primaryKey?: string;
+
+  /**
+   * Gets a base64-encoded 256-bit primary key for signing and
+   * validating the SAS token.
+   */
+  @secret
+  secondaryKey?: string;
+
+  /**
+   * Gets a string that describes the authorization rule.
+   */
+  @visibility(Lifecycle.Read)
+  keyName?: string;
+
+  /**
+   * Gets the last modified time for this rule
+   */
+  @visibility(Lifecycle.Read)
+  // FIXME: (utcDateTime) Please double check that this is the correct type for your scenario.
+  modifiedTime?: utcDateTime;
+
+  /**
+   * Gets the created time for this rule
+   */
+  @visibility(Lifecycle.Read)
+  // FIXME: (utcDateTime) Please double check that this is the correct type for your scenario.
+  createdTime?: utcDateTime;
```

Comment:
what was the original documented format?

Reply:
It's just two lines above.
```
Gets the created time for this rule
```